src/test/java/com/crowdin/client/ai/AiApiTest.java

```diff
@@ -0,0 +1,53 @@
+package com.crowdin.client.ai;
+
+import com.crowdin.client.ai.model.AiSetting;
+import com.crowdin.client.core.model.PatchOperation;
+import com.crowdin.client.core.model.PatchRequest;
+import com.crowdin.client.core.model.ResponseObject;
+import com.crowdin.client.framework.RequestMock;
+import com.crowdin.client.framework.TestClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPatch;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class AiApiTest extends TestClient {
+    private final Long userId = 1L;
+    private final String settingsApi = this.url + "/users/" + userId + "/ai/settings";
+
+    @Override
+    public List<RequestMock> getMocks() {
+        return Arrays.asList(RequestMock.build(settingsApi, HttpGet.METHOD_NAME, "api/ai/getAiSettingResponse.json"),
+                RequestMock.build(this.url + "/users/" + userId + "/ai/settings", HttpPatch.METHOD_NAME, "api/ai/editAiSettingRequest.json",
+                        "api/ai/getAiSettingResponse.json"));
+    }
+
+    @Test
+    public void getAiSettingTest() {
+        System.out.println("test-url: " + settingsApi);
```

Comment:
You might want to remove this line of code

Reply:
Removed log statments, but tests are failing because received response is null. I was not able to debug this, could you PTAL?